BackToOpener/explainer.md

```diff
@@ -0,0 +1,86 @@
+
+This proposal introduces an opt‑in mechanism that signals the browser to insert the opener as an initial entry in the new tab's history. Additionally, when the user navigates back in the new tab, the UA will automatically return focus to the originating tab and close the new tab, provided the originating tab is still active. This feature enhances user experience by supporting a logical back navigation flow, and reduce tab clutter at the same time.
+
+## User-Facing Problem
+
+The new-tab default of chat interfaces and search engines can frustrate users by causing tab proliferation and making it difficult to return to the original conversation or context. Traditional [guidance](https://www.nngroup.com/articles/new-browser-windows-and-tabs/) warns against breaking the user's flow by opening new tabs. At the same time, modern chat interfaces prioritize multitasking and context preservation, which thus prefers opening in new tab.
+
+This proposal bridges the gap by creating a clear navigation pathway back to the originating tab, ensuring users can effortlessly return without navigating through tabs. It additionally helps reduce tab clutter.
+
+## Goals and Use Cases
+
+The primary goal is to allow web developers to maintain a connected navigation experience, where the source context is always within reach even when content is loaded in new tabs. Key use cases include:
+
+- Search Engines: Users can click on search results and then return to their search results page.
+- Conversational Chat Interfaces: Users can click on links and navigate back to the conversation window.
+
+## Proposed Approach
+
+Developers can signal their intent via `window.open()` and `<a>` elements. The browser will then handle the navigation logic, ensuring that when the user navigates back in the new tab, it automatically returns focus to the originating tab and closes the new tab if the opener is still active.
+
+- For `window.open()`, we propose introducing a new `windowFeatures` parameter called `addOpenerToHistory`. When this feature is specified, the browser will add the opener to the new tab's history.
+
+```javascript
+window.open("https://www.destination.com", "_blank", "addOpenerToHistory")
+```
+
+- For `<a>` elements, we propose introducing a new `rel` attribute value called `addOpenerToHistory`. When this value is specified, the browser will add the opener to the new tab's history.
+
+```html
+<a href="https://www.destination.com" target="_blank" rel="addOpenerToHistory">Example Destination</a>
+```
+
+### Observed Behavior
+
+- Upon clicking the back button in the destination tab, if the opener tab is still active, the browser will automatically return focus to the opener tab and close the new tab.
+- If the opener tab is unavailable (e.g., closed or navigated away), the new tab will navigate back to the opener's URL in the current tab.
+
+
+## Alternatives Considered
```

Comment:
did we consider a new `target`? IE something like `target=_blankWithHistory`?

Reply:
Adding a new value for `target` would make this much harder for a site to opt into using because it wouldn't be feature detectable and would degrade in other browsers.

If there was a preference to open in a new window only if this capability was present and otherwise not open in a new window, that would perhaps be viable because of how non-supporting browsers would interpret it.